### PR TITLE
bugfix checkMissingsByValues: reporting of missing tags

### LINKED ISSUE #32: fix checkMissingsByValues() reporting
Currently `checkMissingsByValues()` reports missing tags outside of the specified value range incorrectly.

### PR DESCRIPTION
**Pull request by beckerbenj**
This fixes #32.

### CODE REVIEW

File: R/checkMissings.R [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+#' \code{checkMissingsByValues()} compares labeled values (\code{value}) and missing tags (\code{missings}) of a \code{GADSdat} object
 #' and its meta data information.
 #' Mismatches are reported and can be automatically adjusted. Note that all checks are only applied to the
 #' meta data information, not the actual data. For detecting missing value labels, see \code{\link{checkMissingValLabels}}.
 #'
 #'@param GADSdat \code{GADSdat} object imported via \code{eatGADS}.
 #'@param missingLabel Single regular expression indicating how missing labels are commonly named in the value labels.
 #'@param missingValues Numeric vector of values which are commonly used for missing values.
-#'@param addMissingCode If \code{TRUE}, missing codes are added according to  \code{missingLabel} or \code{missingValues}.
+#'@param addMissingCode If \code{TRUE}, missing tags are added according to  \code{missingLabel} or \code{missingValues}.
```
Comment (franikowsp) [inline R/checkMissings.R:17]: Nit: There is a superfluous space here:

`to _\code{missingLabel}`

File: tests/testthat/test_checkMissings.R [hunk 2]
```diff
@@ -36,8 +36,11 @@ test_that("checkMissings: Missing codes are correctly checked and added", {
 })
 
 test_that("checkMissingsByValues: Missing codes are correctly checked and added", {
-  all_messages <- capture_messages(out <- checkMissingsByValues(df4))
+  all_messages <- capture_messages(out <- checkMissingsByValues(df4, missingValues = -50:-99))
```
Comment (franikowsp) [inline tests/testthat/test_checkMissings.R:39]: Did not know about `capture_messages()` - looks super useful 👍

File: R/checkMissings.R [hunk 3]
```diff
@@ -82,7 +82,7 @@ checkMissingsByValues.GADSdat <- function(GADSdat, missingValues = -50:-99, addM
   labels <- GADSdat$labels
 
   missCode_missing_rows <- which(labels$value %in% missingValues & (is.na(labels$missings) | labels$missings == "valid"))
-  missCode_toomuch_rows <- which(labels$missings == "miss" & labels$value %in% missingValues)
+  missCode_toomuch_rows <- which(labels$missings == "miss" & !labels$value %in% missingValues)
```
Comment (franikowsp) [inline R/checkMissings.R:85]: LGTM

**Review by franikowsp — APPROVED** [review]
Looks all good to me - only one tiny nit.

File: R/checkMissings.R (hunk 1 above)
Comment (beckerbenj) [reply R/checkMissings.R:17]: Eagle eye! 🦅

**Pull request merged**